packages/client/mutations/toasts/mapRequestToJoinOrgToToast.ts

```diff
@@ -25,7 +28,9 @@ const mapRequestToJoinOrgToToast = (
     action: {
       label: 'Review',
       callback: () => {
-        // TODO: Implement review window and add segment events
```

Comment:
+1 I assume we're adding tracking in another PR once we finish the overall implementation?  I'd like to see something like a "Join Request Reviewed" event with an "Action" parameter ("Deny", "Add"), and maybe a list of teams the user was added to?

Reply:
Thanks, I will add this in next PR #8160